common/flatpak-transaction.c

```diff
@@ -881,6 +1072,14 @@ flatpak_transaction_set_disable_dependencies (FlatpakTransaction *self,
   priv->disable_deps = disable_dependencies;
 }
 
+/**
+ * flatpak_transaction_set_disable_related:
+ * @self: a #FlatpakTransaction
+ * @disable_related: whether to avoid adding related refs
+ *
+ * Sets whether the transaction should avoid adding related refs
+ * when resolving operations.
```

Comment:
Maybe we should mention what related refs are. I.e. extensions suggested by the app such as locales.

Reply:
and that